### PR TITLE
separate out the table for clearer directions

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -14,11 +14,55 @@ In this article you will:
...
+
+   `Set ClientSemVer=1.1.0`
+
+4. Add client secrets (or your own config file) to the Configuration.json file
+
+   `"B2C_Client_ClientSecret": "sercret_goes_here",`
+
+   `"AAD_Client_ClientSecret": "sercret_goes_here",`
+  
+   `"AAD_WebApi_ClientSecret": "sercret_goes_here"`
```
Comment (pmaytak) [inline TESTING.md:46]:
```suggestion
   `"B2C_Client_ClientSecret": "secret_goes_here",`

   `"AAD_Client_ClientSecret": "secret_goes_here",`
  
   `"AAD_WebApi_ClientSecret": "secret_goes_here"`
```

File: TESTING.md [hunk 2]
```diff
@@ -28,78 +72,33 @@ In a Developer Command Prompt:
...
-   `cd ProjectTemplates`
-
-4. Add client secrets to the Configuration.json file
+3. Add client secrets to the Configuration.json file
 
    `"B2C_Client_ClientSecret": "sercret_goes_here",`
 
    `"AAD_Client_ClientSecret": "sercret_goes_here",`
   
    `"AAD_WebApi_ClientSecret": "sercret_goes_here"`
```
Comment (pmaytak) [inline TESTING.md:81]:
```suggestion
   `"B2C_Client_ClientSecret": "secret_goes_here",`

   `"AAD_Client_ClientSecret": "secret_goes_here",`
  
   `"AAD_WebApi_ClientSecret": "secret_goes_here"`
```

**Review by pmaytak — APPROVED** [review]
👍

File: TESTING.md [hunk 3]
```diff
@@ -14,11 +14,55 @@ In this article you will:
...
+
+4. Add client secrets (or your own config file) to the Configuration.json file
+
+   `"B2C_Client_ClientSecret": "sercret_goes_here",`
+
+   `"AAD_Client_ClientSecret": "sercret_goes_here",`
+  
+   `"AAD_WebApi_ClientSecret": "sercret_goes_here"`
+
+5. Build the repo. Copy the NuGet package containing the templates (Microsoft.Identity.Web.ProjectTemplates.version.nupkg) downloaded from the release build and paste it under the `ProjectTemplates\bin\Debug` folder of the repo.
```
Comment (jmprieur) [inline TESTING.md:48]: Do we need to build the repo?

File: TESTING.md [hunk 4]
```diff
@@ -14,11 +14,55 @@ In this article you will:
   - Generate C# projects corresponding to all the templates in various configurations (no auth, single-org, single-org calling graph, single-org calling web API, Individual B2C, Individual B2C calling web API (for the web API and the Blazorwasm hosted templates, as B2C does not support OBO)).
   - Configure the projects with existing Azure AD and B2C apps and client secrets. This is done by a configuration file named `configuration.json`. You will need to add the client secrets (see below).
   - Build the generated projects (which are grouped in a solution named `test.sln`).
-- Manually test (for now) the generated projects.
+- [Manually test (for now) the generated projects](How-to-test-the-configured-projects-manually).
 
-## How to generate the test projects
+## How to generate the test projects from a NuGet Package
```
Comment (jmprieur) [inline TESTING.md:19]: We might want to add a sentence saying that it's possible to generate test projects from 2 sources:
- the repo (link to the section below)
- a NuGet package created from a release build.

?

**Review by jmprieur — APPROVED** [review]
LGTM.
Thanks @jennyf19 

I agree that this is better in 2 sections.

File: TESTING.md (hunk 3 above)
Comment (jennyf19) [reply TESTING.md:48]: yes because otherwise there won't be a bin\debug folder

**Pull request merged**